stellarphot/gui_tools/comparison_functions.py

```diff
@@ -484,11 +507,14 @@ def _make_tess_save_box(self):
         self.tess_save_toggle = ipw.ToggleButton(
             description="TESS files...", disabled=True
         )
+        self.tess_save_toggle.layout.visibility = "hidden"
         self._tess_save_box = ipw.VBox()
         self._tess_save_box.layout.visibility = "hidden"
 
+        tess_scope = self.observatory.TESS_telescope_code if self.observatory else None
+        tess_scope = "" if tess_scope is None else tess_scope
         scope_name = ipw.Text(
-            description="Telescope code", value="Paul-P-Feder-0.4m", style=DESC_STYLE
+            description="Telescope code", value=tess_scope, style=DESC_STYLE
```

Comment:
Finally!

Reply:
You have no idea the amount of pain in setting up a form for the observatory 😬 😀